src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -218,6 +226,13 @@ export function transform(formConfig, form) {
       ),
     );
 
+  const addBackRatedDisabilities = formData =>
+    savedRatedDisabilities
```

Comment:
What do you think about making this `savedRatedDisabilities && !formData.ratedDisabilities` so we're not overriding what may already be there?

I can see it going either way, mind you. :man_shrugging: